public/stylesheets/site/2.0/13-group-blurb.css

```diff
@@ -383,7 +386,8 @@ blurbs on the manage collection items pages, mostly reseting styles inherited fr
 }
 
 .bookmark .recent .index {
-    width: 100%;
+  float: none;
```

Comment:
`float:none` is also inherited from:

https://github.com/otwcode/otwarchive/blob/5c610798f66ed4466f6a7eccca8d6205dc2d0d98/public/stylesheets/site/2.0/26-media-narrow.css#L109-L115

unless you want to explicitly have it here.

Reply:
Yeah, we need it in all widths, or else `<ul class="bookmark index group">` ends up escaping `<div class="recent module group">` (so if you, for example, put a blue background on the div, you won't see it because the div's not actually holding the ul)